lib/elasticache_redis.py

```diff
@@ -0,0 +1,599 @@
+"""
+    https://aws.amazon.com/blogs/database/monitoring-best-practices-with-amazon-elasticache-for-redis-using-amazon-cloudwatch/
+"""
+
+from typing import List
+from grafanalib.core import (
+    Alert,
+    AlertCondition,
+    Dashboard,
+    LowerThan,
+    Graph,
```

Comment:
The dashboard is not generated with the name specified in the cli. When I specified the name as `foo` a dashboard with the name `ElastiCache Redis` was generated.


```bash
 python3 bin.py --name foo --environment qa --cloudwatch_data_source cloudwatch elasticache-redis --cache_cluster_id my-comtravo-qa
```